packages/pds/src/image/sharp.ts

```diff
@@ -0,0 +1,79 @@
+import { Readable } from 'stream'
+import sharp from 'sharp'
+import {
+  formatsToMimes,
+  forwardStreamErrors,
+  ImageProcessor,
+  Options,
+} from './util'
+
+export class SharpImageProcessor implements ImageProcessor {
```

Comment:
since this doesn't wrap any sort of state, thoughts on splitting it up & just making it two functions?

Reply:
Yeah for sure, good call 👍 originally was thinking `ImageProcessor` would be a pluggable interface that would have to be passed into the server, but I think it's safe to say that's become a non-concern.